src/coprocessor/dag/batch/executors/slow_hash_aggr_executor.rs

```diff
@@ -449,27 +449,27 @@ mod tests {
             .unwrap();
         assert_eq!(
             r.physical_columns[3].decoded().as_int_slice(),
-            &[Some(5), None, None, Some(1)]
+            &[Some(5), Some(1), None, None]
```

Comment:
Why does the ordering change if we are using the same hasher as before?

Reply:
@nrc Not sure, I tried reading through the implementation of `Hashbrown` and `std::HashMap` but couldn't find the underlying reason. The `std::HashMap` uses hashbrown as base and I couldn't find any implementation difference to cause this change.

Reply:
Fair enough, thanks for looking into it.